crates/evm/evm/src/inspectors/stack.rs

```diff
@@ -368,23 +365,12 @@ impl<DB: DatabaseExt> Inspector<DB> for InspectorStack {
                 &mut self.printer
             ],
             |inspector| {
-                let status = inspector.initialize_interp(interpreter, data);
-
-                // Allow inspectors to exit early
```

Comment:
Maybe here? Add back early exit check on `interpreter`

Reply:
there's no return value anymore, I must have overlooked something else

Reply:
ah yeah, this would mirror the previous behaviour